docs/platform/howto/byoc/networking-security.md

```diff
@@ -0,0 +1,123 @@
+considered dynamic for the purposes of firewall policies. To that end, an egress of
+`0.0.0.0/0` is generally required to accommodate traffic to Aiven Management.
+:::
+
+## Workload nodes networking
+
+Management traffic to and from workload nodes is sent through the bastion proxy and
+encrypted. Aiven Management and API calls are always encrypted. SSH access is permitted if
+Aiven support staff require direct access to the bastion or workload nodes. SSH
+connections from support staff are logged, monitored, and require validation by Aiven’s
+security operations team. SSH connections originate from Aiven Management plane's NAT gateways.
+
+### Traffic to the workload
+
+On top of SSH and Aiven Management Plane connectivity, there are also ingress and egress
+rules from the Customer Networks that need to be defined to allow customer applications to
+connect to Aiven services. This depends on the cloud provider being used, and the
+customer’s network architecture.
+
+| Source                   | Destination     | Description                                                                                                                             |
+| ------------------------ | ----------------| --------------------------------------------------------------------------------------------------------------------------------------- |
+| Bastion subnet           | Workload subnet | Aiven uses SSH from the bastion subnet to the workload subnet for troubleshooting, setup, or configuration tasks on the workload nodes. |
+| Bastion subnet           | Workload subnet | Aiven Management plane accesses the workload nodes via the bastion node to retrieve status information and perform routine operations.  |
+| Your remote applications | Workload subnet | Varies depending on services being used.                                                                                                |
+
+### Traffic from the workload
+
+The workload subnet nodes communicate with each other. The nature of these communications
+varies depending on the services, features, and plugins being used.
+
+| Source          | Destination              | Description                                                                                                                |
+| --------------- | -------------------------| -------------------------------------------------------------------------------------------------------------------------- |
+| Workload subnet | Aiven Management         | Communication channel with the Aiven Management plane used to collect metrics and logs from the bastion and workload nodes |
+| Workload subnet | Aiven Management         | Required for the bastion and workload nodes to make calls to the Aiven Management plane                                    |
+| Workload subnet | AWS CloudFront           | Communication channel for downloading RPM packages for bastion and workload nodes setup                                    |
```

Comment:
Traffic from the workload subnet to CloudFront is only relevant to the workload nodes:
```suggestion
| Workload subnet | AWS CloudFront           | Used for downloading RPM packages for workload nodes setup                                    |
```